components/Article/Page.js

```diff
@@ -235,7 +245,10 @@ const ArticlePage = ({
                   })
                 }
               })
-            : undefined
+            : undefined,
+        withCommentData:
+          process.browser && window ? withCommentData : undefined,
```

Comment:
if the graphql wrapper has `ssr: false`—which it seems to have via styleguide—you shouldn't need that `process.browser` or window check. Also if you want to check a window (process.browser is webpack specific but fine in next.js code bases) you need to do `typeof window !== 'undefined'` just `window` will already trigger an exception in node.

Reply:
will change. the problem also arises in publikator, though.